src/lib/components/Funding/FunderDropdown.js

```diff
@@ -11,69 +11,74 @@ import { Dropdown } from 'semantic-ui-react';
 import { withState } from 'react-searchkit';
 import { i18next } from '@translations/i18next';
 
-export const FunderDropdown = withState(({ currentResultsState: awardsList, updateQueryState: updateQueryState, currentQueryState: currentQueryState }) => {
+export const FunderDropdown = withState(
+  ({
+    currentResultsState: awardsList,
+    updateQueryState: updateQueryState,
+    currentQueryState: currentQueryState,
+  }) => {
     const [fundersFromFacets] = useFundersFromFacets(awardsList);
-  
+
     /**
      * Trigger on funder selection.
      * Updated the query state to filter by the selected funder.
-     * 
-     * @param {*} event 
-     * @param {*} data 
+     *
+     * @param {*} event
+     * @param {*} data
      */
     function onFunderSelect(event, data) {
       let newFilter = [];
-  
-      if (data && data.value !== "") {
-        newFilter = ['funders', data.value]
+
+      if (data && data.value !== '') {
+        newFilter = ['funders', data.value];
       }
-      updateQueryState({...currentQueryState, filters: newFilter, page: 1 });
+      updateQueryState({ ...currentQueryState, filters: newFilter, page: 1 });
     }
-  
+
     /**
      * Custom hook, triggered when the awards list changes.
      * It retrieves funders from new award's facets.
-     * 
-     * @param {object} awards 
-     * 
+     *
+     * @param {object} awards
+     *
      * @returns {object[]} an array of objects, each representing a facetted funder.
      */
     function useFundersFromFacets(awards) {
       const [result, setResult] = React.useState([]);
       React.useEffect(() => {
-  
         /**
-        * Retrieves funders from awards facets and sets the result in state 'result'.
-        */
+         * Retrieves funders from awards facets and sets the result in state 'result'.
+         */
         function getFundersFromAwardsFacet() {
           if (awards.loading) {
             setResult([]);
             return;
           }
-  
-          const funders = awards.data.aggregations?.funders?.buckets.map((agg) => {
-            return {
-              key: agg.key,
-              value: agg.key,
-              text: agg.label
-            };
-          })
+
+          const funders = awards.data.aggregations?.funders?.buckets.map(
+            (agg) => {
+              return {
+                key: agg.key,
+                value: agg.key,
+                text: agg.label,
+              };
+            }
+          );
           setResult(funders);
         }
-  
+
         getFundersFromAwardsFacet();
       }, [awards]);
-  
+
       return [result];
     }
-  
+
     return (
       <Dropdown
         placeholder={i18next.t('Funder')}
         search
         selection
         clearable
-        scrolling
```

Comment:
how does it display with removed scrolling and large number of results ?

Reply:
It still displays a scroll (I am testing with mocked data)

![image](https://user-images.githubusercontent.com/15194802/171160600-7579c21e-569a-4822-87e1-0a1634576001.png)

with `scrolling` props the dropdown was not so big, but if we add it, it will cause the missalignment, another option would be to set a max-height using css instead of the `scrolling` props

Reply:
I think it is better to find the cause of the misalignment and fix the css while using scrolling, rather to find workarounds on how not to use scrolling :) 
think about this in terms of someone else implementing another new dropdown, and reintroducing the misalignment yet again. Isn't it better to fix it once for all ?

Reply:
Well, I found the cause of the issue but I am not sure it's us ... Actually in a first look I would say it's a semantic ui issue and fixing it will probably imply to change the css when the `scrolling` prop is applied. I am saying that the error is on their side because when applying the scrolling, we also apply the following `min-width: 100% !important;` (see [code here](https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/modules/dropdown.less#L1081)) and this overrides the folldowing rule ` min-width: calc(100% + 2px );` (see [code here](https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/modules/dropdown.less#L419)). We need that extra 2px to avoid the missalignment. That said if we agree, I can override the `.ui.selection.dropdown .menu`  to have the same correct width, but again, we are introducing custom css, that's what I was trying to avoid by removing the `scrolling` prop.

Reply:
it is totally OK to introduce custom CSS if it is fixing a global issue - that's why we have a centrally managed theming. The problem with custom CSS appears when it is duplicated across different components or created to "reinvent the wheel" while the main theme already provides the needed styling

Reply:
Added back the scrolling and fixed the rule in invenio-theme, [here the PR.](https://github.com/inveniosoftware/invenio-theme/pull/290)